### PR TITLE
Remove cleanup from build action

### PR DESCRIPTION
**Pull request by garethbowen**
Instead add it as a separate build that can be executed overnight.

medic/cht-core#7025

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by garethbowen** [comment]
@dianabarsan I hacked this up quickly. I don't know how to test if it works or not, but maybe it'll run overnight tonight? Anyway, can you please review the concept and see what you think?

**Review by dianabarsan — APPROVED** [review]
Thanks for picking this up! This solution works for me, I actually prefer it. 

I wonder if the action will trigger if it's in a PR, and not committed to `master`. 
Would that mean that anyone submitting a PR with a schedule GHA workflow will get a chance to trigger this action on OUR account?

File: .github/workflows/cleanup.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: Build cht-core and test against node versions
```
[the remark this answers is not in the document]
Comment (garethbowen) [reply .github/workflows/cleanup.yml]: :facepalm: Done.

**Comment by garethbowen** [comment]
From [the documentation](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onschedule):

> Scheduled workflows run on the latest commit on the default or base branch.

Which makes sense, so I think we're good.

**Comment by garethbowen** [comment]
Merged so it'll run. I'll check back tomorrow to make sure it works.

**Comment by dianabarsan** [comment]
It worked! https://github.com/medic/cht-core/actions/runs/681713218

**Pull request merged**